spec/features/idv/threatmetrix_pending_spec.rb

```diff
@@ -49,4 +63,69 @@
 
     expect(current_path).to eq('/auth/result')
   end
+
+  scenario 'users threatmetrix Pass, it logs idv_tmx_fraud_check event', :js do
+    freeze_time do
+      complete_all_idv_steps_with(threatmetrix: 'Pass')
+      expect_irs_event(expected_success: true, expected_failure_reason: nil)
+    end
+  end
+
+  scenario 'users pending threatmetrix Reject, it logs idv_tmx_fraud_check event', :js do
+    freeze_time do
+      expect_pending_failure_reason(threatmetrix: 'Reject')
+    end
+  end
+
+  scenario 'users pending threatmetrix Review, it logs idv_tmx_fraud_check event', :js do
+    freeze_time do
+      expect_pending_failure_reason(threatmetrix: 'Review')
+    end
+  end
+
+  scenario 'users pending threatmetrix No Result, it logs idv_tmx_fraud_check event', :js do
+    freeze_time do
+      expect_pending_failure_reason(threatmetrix: 'No Result')
+    end
+  end
+
+  def complete_all_idv_steps_with(threatmetrix:)
```

Comment:
This method could be moved to `/spec/support/features/doc_auth_helper.rb` - looks like it might be useful for more specs in the future.